ops/cli.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash -ex
+
+# print with style
+# all available styles:
+# info | error | success | warn | debug
+function print_style {
+    if [ "$1" == "info" ];
+    then
+        # print gray
+        printf '\e[1;90m%-6s\e[m\n' "$2"
+    elif [ "$1" == "error" ];
+    then
+         # print red
+        printf '\e[1;91m%-6s\e[m\n' "$2"
+    elif [ "$1" == "success" ];
+    then
+        # print green
+        printf '\e[1;92m%-6s\e[m\n' "$2"
+    elif [ "$1" == "warn" ];
+    then
+        # print yellow
+        printf '\e[1;93m%-6s\e[m\n' "$2"
+    elif [ "$1" == "debug" ];
+    then
+        if [ "${DEBUG}" == "true" ];
+        then
+            # print gray
+            printf '\e[1;90m%-6s\e[m\n' "$2"
+        fi
+    else
+        printf "%s\n" "$1"
+    fi
+}
+
+# check if the command is available in the shell
+function check_availability {
+    local some_command=$1
+
+    hash "$some_command" 2>/dev/null || { 
+        print_style >&2 "error" "${some_command} is required but not found! Aborting..."; 
+        exit 1; 
+    }
+}
+
+# draw a line of char
+function line {
+    local char=${1:-"-"}
+    printf %"$(tput cols)"s |tr " " "${char}"
+}
+
+# render Helm values
+function render_helm_values {
+    local CTMPL_FILE=$1
+
+    docker run -i \
+               --rm \
+               -v "$(pwd)":/working \
+               -v "${HOME}"/.vault-token:/root/.vault-token \
+               broadinstitute/dsde-toolbox:dev \
+               /usr/local/bin/render-ctmpls.sh \
+               -k "${CTMPL_FILE}"
+}
+
```

Comment:
Did you intend `CHART_REPO_ALIAS` to be local? (It isn't used here.)
Or do you expect its value to show up in `deploy_helm_charts` too?

Reply:
Fixed